lib/src/clojure_lsp/feature/stubs.clj

```diff
@@ -25,7 +26,7 @@
 
 (defn ^:private generate-stubs! [namespaces settings db]
   (try
-    (if-let [classpath (string/join ":" (:classpath db))]
+    (if-let [classpath (string/join fs/path-separator (:classpath db))]
```

Comment:
I didn't know windows used different path-separator for classpath, good to know

Reply:
It's great to have a comprehensive test suite to catch these cases :)